functions/src/index.ts

```diff
@@ -39,10 +39,7 @@ export * from './functions/customSeed.js'
 export * from './functions/defaultSeed.js'
 export * from './functions/dismissMessage.js'
 export * from './functions/getUsersInformation.js'
-export * from './functions/onHistoryWritten.js'
 export * from './functions/onSchedule.js'
-export * from './functions/onUserDocumentWritten.js'
-export * from './functions/onUserWritten.js'
```

Comment:
You may possibly need to ask Garrick to remove those functions before trying to deploy